lib/graphql/fragment_cache/schema/tracer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module GraphQL
+  # Plugin definition
+  module FragmentCache
+    module Schema
+      class Tracer
+        using Ext
+
+        class << self
+          def trace(key, data)
+            yield.tap do |resolved_value|
+              next unless connection_to_cache?(key, data)
+
+              # We need to attach connection object to fragment and save it later
+              context = data[:query].context
+              verify_connections!(context)
+              cache_connection(resolved_value, context)
+            end
+          end
+
+          private
+
+          def connection_to_cache?(key, data)
+            key == "execute_field" && data[:field].connection?
+          end
+
+          def verify_connections!(context)
+            return if context.schema.new_connections?
+
+            raise StandardError,
+              "GraphQL::Pagination::Connections should be enabled for connection caching"
```

Comment:
Can we add a link to the docs/guide on what is it and how to enable it?

Reply:
There is a [mention](https://github.com/DmitryTsepelev/graphql-ruby-fragment_cache/blame/master/README.md#L64) in the README